apps/fishing-map/features/datasets/datasets.selectors.ts

```diff
@@ -25,6 +25,16 @@ const selectDatasetsByType = (type: DatasetTypes) => {
 export const selectFourwingsDatasets = selectDatasetsByType(DatasetTypes.Fourwings)
 export const selectVesselsDatasets = selectDatasetsByType(DatasetTypes.Vessels)
 
+export const selectVesselGroupCompatibleDatasets = createSelector(
+  [selectVesselsDatasets],
+  (datasets) => {
+    return datasets.filter(
+      (d) =>
+        d.status !== DatasetStatus.Deleted && d.configuration?.apiSupportedVersions?.includes('v3')
```

Comment:
Maybe would be better to create a constant to store the current supported version `v3` so whenever an API migration is needed we can just switch the version in the configuration?